content_schemas/formats/shared/definitions/_logo.jsonnet

```diff
@@ -0,0 +1,32 @@
+{
+  type: "object",
```

Comment:
Are any of the properties here required?

Reply:
Looks like the title is required, but the crest and image are optional.  I'll add this as required.